tests/v2/validation/provisioning/airgap/README.md

```diff
@@ -11,17 +11,17 @@ provisioningInput:
   - calico
   # For the kubernetes versions please set it to what is appropriate for that release check
   k3sKubernetesVersion:
-  - v1.23
-  - v1.24
-  - v1.25
+  - v1.30
```

Comment:
Is there a need to have this? I think in our READMEs, constantly updating k8s versions becomes a bit of a pain.

Reply:
not necessary to me either, but can we remove in a separate PR?

Reply:
Yeah that's fine, we can do later 👍🏿